app/views/dashboard.html

```diff
@@ -98,8 +98,8 @@ <h4 translate="legend.ptratio">Pupil-Teacher Ratio</h4>
     <div class="row map-widget-row">
       <div class="col-sm-12 map-container">
         <ul class="nav nav-tabs" id="nav-tabs-dashboard">
-          <li role="presentation" ng-class="{active: viewMode==='regional'}">
-            <a ng-click="setViewMode('regional')">
+          <li role="presentation" ng-class="{active: viewMode==='regions'}">
+            <a ng-click="setViewMode('regions')">
```

Comment:
Right now you can click on "Primary Schools" again to get back to the school view (if you're looking at primary schools). We can make this button a toggle as well if you want.